grpc/go-cubic/cubicpb/gw/go.mod

```diff
@@ -4,9 +4,19 @@ go 1.12
 
 require (
 	github.com/cobaltspeech/sdk-cubic/grpc/go-cubic v1.6.3
+	github.com/desertbit/timer v0.0.0-20180107155436-c41aec40b27f // indirect
```

Comment:
Can we move all of the Go changes into a different PR?  Or was this needed to support the new improbable-eng/grpc-web stuff?  

If it's needed, would it be better to put that into a separate PR, merge it, and then add node support in a second PR?

Reply:
https://github.com/cobaltspeech/sdk-cubic/pull/111